app-frontend/src/app/components/projects/projectItem/projectItem.html

```diff
@@ -1,14 +1,51 @@
 <div class="panel-body">
-  <div class="tag-container">
-    <h4 class="project-title">{{$ctrl.project.name}}</h4>
-    <rf-status-tag ng-if="$ctrl.status" entity-type="project" status="$ctrl.status"></rf-status-tag>
+  <rf-status-tag ng-if="$ctrl.status" entity-type="project" status="$ctrl.status"></rf-status-tag>
+  <a href ui-sref="projects.detail({projectid: $ctrl.project.id})" class="h4">
+    <h4 class="project-title" data-title="{{$ctrl.project.name}}">{{$ctrl.project.name}}</h4>
+  </a>
+  <div class="row">
+    <div class="column-12">
+
+      <!-- IF project has no scenes -->
+      <div class="project-preview-container placeholder" 
+           ng-if="!$ctrl.project.scenes && $ctrl.status !== 'CURRENT'"
+           style="background-image: url({{$ctrl.projectPlaceholder}})">
+        <span>This project doesn't have any scenes yet</span>
+      </div>
+
+       <!-- IF no ingested scenes: -->
```

Comment:
I talked to Chris this morning and he explained that we create the thumbnails on the fly when they request the project page. We could choose to display a thumbnail while scenes are ingesting, but because the thumbnails are still janky Chris would prefer not to.